styles.css

```diff
@@ -0,0 +1,148 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  justify-content: center;
+  background-color: #F9F9F9;
+  }
+
+.nav-bar {
+  display: inline-flex;
+  justify-content: space-between;
+  width: 100%;
+  position: top;
+  align-items: center;
+  }
+
+  #nav1 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  #nav2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+  }
+
+  #search-bar {
+    width: 700px;
+  }
+
+  #search-button {
+    width:10%;
+  }
+
+#img-logo {
+  height: 60%;
```

Comment:
Be consistent with your spaces and tabs. Kindly fix this section and other sections of your CSS rules that are not well indented